### ISSUE TITLE
Performance difference between v0.12.14 and v0.13.1

### ISSUE THREAD

**Issue by adanud** [openpifpaf/openpifpaf#620]
After updating openpifpaf to newer version there is noticeable difference in prediction output.
Two versions are compared, the v0.12.14 and v0.13.1 using the shufflenetv2k16-wholebody model. Printing the openpifpaf.network.factory.CHECKPOINT_URLS['shufflenetv2k16-wholebody'] confirmed that the same trained model weights are used. The older version seems to have better prediction while the newer version produced empty prediction.

What could be causing this issue? Can there be a way to improve the prediction with the new version?

Here are the results:
![image](https://github.com/openpifpaf/openpifpaf/assets/154599038/79b5451c-96b9-4ff7-95df-34730c781d49)
![image](https://github.com/openpifpaf/openpifpaf/assets/154599038/c38fc1cc-f89c-44f9-9f61-72c905025ace)

The same result as v0.13.1 is obtained from today's main branch.
![image](https://github.com/openpifpaf/openpifpaf/assets/154599038/a5d1aed4-dad8-4c0a-a184-e8de23422a13)

Here is the code that is used to run the prediction:

```
import openpifpaf
import torch
import cv2
import os
import matplotlib.pyplot as plt
%matplotlib inline
from openpifpaf.plugins.wholebody import constants as wb_constants

def draw_skeleton(ann, ax, w = 3):
    for i, kp in enumerate(wb_constants.WHOLEBODY_SKELETON):
        kp = [_-1 for _ in kp]
        if kp[0] < ann.data.shape[0] and kp[1] < ann.data.shape[0]:
            if ann.data[kp[0],2] > 0 and ann.data[kp[1],2] > 0:
                if kp[0] < 91 or kp[0] > 132:
                    color = matplotlib.cm.get_cmap('tab20')((i % 20 + 0.05) / 20)
                else:
                    if 111 < kp[0]:
                        color = 'r'
                    else:
                        color = 'g'
                ax.plot( [ann.data[kp[0],0], ann.data[kp[1],0] ], [ann.data[kp[0],1], ann.data[kp[1],1] ], color=color, linewidth=w)



img_file = r'image.jpeg' # from https://images.pexels.com/photos/4009745/pexels-photo-4009745.jpeg

img = cv2.cvtColor(((cv2.imread(img_file))), cv2.COLOR_BGR2RGB)
scale = 15
img = cv2.resize(img, tuple([ int(_ / scale) for _ in img.shape[:2][::-1] ]))

if openpifpaf.__version__=='0.13.1':
    CppCifCaf = torch.classes.openpifpaf_decoder.CifCaf 
    CppCifCaf.set_force_complete(True)
elif openpifpaf.__version__=='0.12.14':
    openpifpaf.decoder.CifCaf.force_complete = True
    
checkpoint = 'shufflenetv2k16-wholebody'
predictor = openpifpaf.Predictor(checkpoint=checkpoint,visualize_image=False)

prediction,_,meta = predictor.numpy_image(img)    

plt.figure(figsize=(5,5))

plt.imshow(img)
ax = plt.gca()
for r in prediction:
    draw_skeleton(r,ax,w=5)
plt.axis('off')
plt.title('Openpifpaf v{} Detection'.format(openpifpaf.__version__))

plt.tight_layout()
```
